clients/web/src/components/content-ads/content-ads.js

```diff
@@ -16,88 +42,64 @@ const adRailStyle = css`
     margin-top: -22px;
   }
 `
+
+/**
+ * Suprisingly, this defines styles for ads below the fold
+ */
 const belowTheFoldStyle = css`
   margin-top: 4rem;
 `
 
-// Syndicated Article MozAds placements
-const ABOVE_THE_FOLD_MARS_ID = "pocket_billboard_1"
-const BELOW_THE_FOLD_MARS_ID = "pocket_billboard_2"
-const ABOVE_THE_FOLD_MOBILE_MARS_ID = "pocket_mrec_1"
-const BELOW_THE_FOLD_MOBILE_MARS_ID = "pocket_mrec_2"
-const RIGHT_RAIL_1_MARS_ID = "pocket_skyscraper_1"
-const RIGHT_RAIL_2_MARS_ID = "pocket_skyscraper_2"
-
-let placements = {}
-placements[ABOVE_THE_FOLD_MARS_ID] = IABFixedSize.Billboard,
-placements[BELOW_THE_FOLD_MARS_ID] = IABFixedSize.Billboard,
-placements[ABOVE_THE_FOLD_MOBILE_MARS_ID] = IABFixedSize.MediumRectangle
-placements[BELOW_THE_FOLD_MOBILE_MARS_ID] = IABFixedSize.MediumRectangle
-placements[RIGHT_RAIL_1_MARS_ID] = IABFixedSize.Skyscaper,
-placements[RIGHT_RAIL_2_MARS_ID] = IABFixedSize.Skyscaper
+/**
+ * BillboardAboveTheFold
+ * ---
+ * Full size horizontal ad
+ * This ad collapses so there isn't a big screen jump since it sits above the content
+ */
+export function BillboardAboveTheFold({ isMobile, allowAds, resolved, targeting }) {
+  const collapse = allowAds && resolved
+  const viewport = useViewport()
+  const showMobile = viewport.width < 600 || isMobile
+  const placement = showMobile ? Placements.ABOVE_THE_FOLD_MOBILE : Placements.ABOVE_THE_FOLD
```

Comment:
Using the viewport provider to determine which ad to show.  I have an open question about this, but I think this is fine in terms of having the side effect of showing more ads.

Reply:
![money-printer-go-brrrrr](https://emoji.slack-edge.com/T07DB2PSS3W/money-printer-go-brrrrr/f154d71ada9d5b59.gif)